src/app/projects/projlist-filters/projlist-filters.component.ts

```diff
@@ -259,6 +303,18 @@ export class ProjlistFiltersComponent implements OnInit, OnChanges, OnDestroy {
       item._id.toString().indexOf(this.dispIdFilter.toString()) > -1
     );
 
+		// If there are any filters checked, check for matching project type filters
+		if (this.projectTypeFilters.find(ptf => true === ptf.checked)) {
```

Comment:
The `filter` and `map` array methods should be able to make lines 307 - 314 into this:
```suggestion
    // Filters by only those that are checked, then only returns the name for each one instead of the object
		const validatedLocations = this.projectTypeFilters.filter(ptf => ptf.checked).map(ptf => ptf.name);
```